api/types.proto

```diff
@@ -489,12 +489,17 @@ message NetworkAttachmentConfig {
 	// Target specifies the target network for attachment. This value must be a
 	// network ID.
 	string target = 1;
+	
 	// Aliases specifies a list of discoverable alternate names for the service on this Target.
 	repeated string aliases = 2;
+	
 	// Addresses specifies a list of ipv4 and ipv6 addresses
 	// preferred. If these addresses are not available then the
 	// attachment might fail.
 	repeated string addresses = 3;
+	
+	//Driver options is a map of driver specific options for the network target
```

Comment:
@aaronlehmann I added spaces that were non existent only for this message in the file to make it consistent.
@stevvooe I don't see a type. And I believe we do pack messages in map. Is there an alternative or any known issues with this approach ?